packages/email-petition/src/email-petition.tsx

```diff
@@ -48,6 +50,8 @@ export function EmailPetition(props: {
   onSubmit?: () => void;
   debug: boolean;
   test: boolean;
+  hasPetitionThermometer: boolean;
+  petitionThermometerGoal?: number;
```

Comment:
this is already the petition component, so calling this "petition" thermometer isn't helpful. "signature thermometer" would be better.

also this smells of "primitive obsession". won't block on it but if you're down, making a small object would help keep args organized as we add more in the future.

also it makes validation easier. with this suggestion it's impossible for developers to set has=true and goal=undefined, for example.

```suggestion
  signatureThermometer?: {
    defaultGoal: number
  }
```

Reply:
Added this, made validation easier as you said! Thanks for the suggestion!!